pyproject.toml

```diff
@@ -38,7 +38,7 @@ python = ">=3.8,<3.13"
 # Background task processing
 celery = "~5.3.1"
 # Fundamental web framework for Nautobot
-Django = "~3.2.25"
+Django = "~4.2"
```

Comment:
We should probably update this to `~4.2.13` since that's the current latest version and versions 4.2.11 and earlier have known CVEs.